specification/network/resource-manager/Microsoft.Network/stable/2020-07-01/loadBalancer.json

```diff
@@ -1383,6 +1386,18 @@
             "name": "LoadBalancerSkuName",
             "modelAsString": true
           }
+        },
+        "tier": {
+          "type": "string",
+          "description": "Tier of a load balancer SKU.",
+          "enum": [
+            "Regional",
```

Comment:
I assume the default value in the server side is `Regional`. Is that correct?

Reply:
Yep